.ci.yaml

```diff
@@ -441,7 +441,7 @@ targets:
       config_name: mac_host_engine
       dependencies: >-
         [
-          {"dependency": "goldctl", "version": "git_revision:f808dcff91b221ae313e540c09d79696cd08b8de"}
```

Comment:
Actually this one might not be necessary at all. This task is just the `engine_v2` orchestrator task (the parent task that spawns the sub-builds). It seems unlikely that any actual tests will run in the orchestrator that use skia gold.